lib/modules/hosts/soundcloud.js

```diff
@@ -8,21 +7,11 @@ export default {
 	logo: 'https://a-v2.sndcdn.com/assets/images/sc-icons/favicon-2cadd14b.ico',
 	detect: () => true,
 	async handleLink(href) {
-		const { html } = await ajax({
-			url: 'https://soundcloud.com/oembed',
-			data: {
-				url: href,
-				format: 'json',
-				iframe: 'true',
-			},
-			type: 'json',
-		});
-
-		// Get src from iframe html returned
-		const src = $(html).attr('src');
+		const src = string.encode`https://w.soundcloud.com/player/?url=${href}`;
 		return {
 			type: 'IFRAME',
 			embed: src,
+			height: '124px',
```

Comment:
I think 166px is better (it's also used [here](https://soundcloud.com/pages/embed)). And `width: '100%'`

Examples:

200px
![image](https://cloud.githubusercontent.com/assets/7673145/15728480/a0bdb200-282a-11e6-88fb-2e4d545ac18b.png)

166px
![image](https://cloud.githubusercontent.com/assets/7673145/15728498/bbc54a36-282a-11e6-9b3e-35b8fff422d8.png)

124px
![image](https://cloud.githubusercontent.com/assets/7673145/15728509/ce52458c-282a-11e6-8e74-3e11fd33619c.png)